packages/mysqli/static.mak

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env make
+
+WITH_MYSQLI?=1
+
+ifeq ($(filter ${WITH_MYSQLI},0 1),)
+$(error WITH_MYSQLI MUST BE 0 OR 1. PLEASE CHECK YOUR SETTINGS FILE: $(abspath ${ENV_FILE}))
+endif
+
+ifeq (${WITH_MYSQLI},1)
+WITH_MYSQLI=static
+endif
+
+ifeq (${WITH_MYSQLI},static)
+CONFIGURE_FLAGS+= --enable-pdo --with-mysqli --with-zlib
```

Comment:
PDO and mysqli are two different ways of accessing the database; PDO supports multiple backends and a common interface for all of them. I think we should follow up to understand if this is enough or if enabling PDO would require us to enable more configure/build settings so that those backends are available to PDO.

Reply:
[I think they are different extensions](https://www.php.net/manual/en/ref.pdo-mysql.php) and `mysqli` is the modern one. I would remove it from here now as I think it requires a separate flag to add it. You usually run one of those extensions, but no both.